### DISCUSSION TITLE
Is it possible to force opening MRs for specific packages ignoring any limits?

### DISCUSSION THREAD

**Discussion by herndlm** [renovatebot/renovate#23028] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

We have some internal packages that I want to keep updated via renovate. And they should always be prioritized over all other packages. Currently our repo has already 5 open MRs (the configured limit), there is an update of an internal package, but Renovate is not opening the MR and lists the update as being "rate-limited". Is there a way to avoid this and force a MR?

I have configured a packageRule already, that should not be overriden any further. Here are the relevant parts of renovate.json:
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    ":automergeMinor",
    ":maintainLockFilesWeekly",
    ":semanticCommitTypeAll(chore)",
    ":semanticCommits",
    "regexManagers:dockerfileVersions"
  ],
  "additionalBranchPrefix": "{{parentDir}}-",
  "branchPrefix": "feature/renovate-",
  "internalChecksAsSuccess": true,
  "onboardingConfigFileName": ".renovaterc",
  "onboardingPrTitle": "feat: configure renovate",
  "prConcurrentLimit": 5,
  "prCreation": "not-pending",
  "rangeStrategy": "update-lockfile",
  "rollbackPrs": true,
  "stabilityDays": 3,
  "platformAutomerge": true,
  "packageRules": [
    ...
    {
      "description": "Enable immediate high-priority handling for acmme updates including major updates",
      "matchPackagePatterns": ["^@acme/", "^acme/", "^acme-plugin/", "^acme-theme/", "^wp-plugin/ace"],
      "groupName": "acme packages",
      "major": {
        "enabled": true
      },
      "branchConcurrentLimit": 0,
      "prConcurrentLimit": 0,
      "prCreation": "immediate",
      "prHourlyLimit": 0,
      "prPriority": 99,
      "stabilityDays": 0
    },
    ...
  ]
}
```

### What you tried so far.

reset any possible config option that I found that deals with limits in our packageRule

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Today:
- Use prPriority to prioritize the order of creation (as you're doing)
- Use dependencyDashboard to force open any pending you want

Future: #10546

**Reply by herndlm** [reply]
I see, thx. Yeah, that's what I'm doing already 😊
